zerver/lib/home.py

```diff
@@ -208,8 +208,14 @@ def build_page_params_for_home_page_load(
         # There is no event queue for spectators since
         # events support for spectators is not implemented yet.
         no_event_queue=user_profile is None,
+        server_sentry_dsn=settings.SENTRY_FRONTEND_DSN,
     )
 
+    if settings.SENTRY_FRONTEND_DSN is not None:
+        page_params["realm_sentry_key"] = realm.string_id
+        page_params["server_sentry_sample_rate"] = settings.SENTRY_FRONTEND_SAMPLE_RATE
+        page_params["server_sentry_trace_rate"] = settings.SENTRY_FRONTEND_TRACE_RATE
```

Comment:
The CI failures are likely because we need to configure `test_home.py` to except these.